tools/accuracy_checker/accuracy_checker/main.py

```diff
@@ -235,6 +235,11 @@ def build_arguments_parser():
         required=False,
         type=cast_to_bool
     )
+    parser.add_argument(
+        '-v', '--vocab', help="Prefix for vocabulary files for NLP models",
```

Comment:
This seems really specific. Why do we need a separate prefix just for this?

Reply:
usuually it is stored with model and when the original model stored separatly with IR we need to copy this file to every directory with converted model (e.g. model package for internal validation, IR cache in OV validation and so on) if we not provide the path to model in config it can become a problem to find vocab

Reply:
OK, but couldn't a more generic solution be implemented? E.g. an ability to add extra paths in which all model-related files would be searched for if they're not found in the main model directory.

Reply:
What do you mean under model-specific files?
It was just fast prototype in format as it was asked in validation,
I want also add opportunity to provide the full path in command line without specification arg in the config, it is the reason why I add this in command line with specific appropriate name

Reply:
> What do you mean under model-specific files?

I never said "model-specific files".

> It was just fast prototype in format as it was asked in validation,

If you want it to be merged, then it's not a prototype, but a complete feature. :-)

> I want also add opportunity to provide the full path in command line without specification arg in the config, it is the reason why I add this in command line with specific appropriate name

I don't understand what you mean by this.

Reply:
I want to merge it in the next release only, so I have a lot of time to rewrite it : )

Reply:
Okay, see you in the next release, then. :-)

(Also, consider opening draft pull requests for changes that are not meant to be merged ASAP.)

Reply:
It was prepared as fast workarouund for validation team, but they found another.
In the next case I take in account it and create draft, of course :)